feat: provide `ctx.signal` (#7878)
vitest-dev/vitest: docs/guide/test-context.md

})
```

#### `context.signal` <Version>3.2.0</Version> {#context-signal}

> A signal object that can be aborted by Vitest. The signal is aborted in these situations:

Reviewer: Maybe:

```suggestion
An [`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) that can be aborted by Vitest. The signal is aborted in these situations:
```
